walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,107 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
+/** Class that lists possible meeting times based on meeting information it takes in. */
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+  private static final int END_OF_DAY = TimeRange.getTimeInMinutes(23, 59);
```

Comment:
This may cause problems because if I asked for a 1-hour meeting and the only open block started at 11PM, it would not be found.

Reply:
l watch for this case by saying it's inclusive instead of exclusive when using END_OF_DAY as the end of a range.